bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php

```diff
@@ -923,39 +923,6 @@ public function updateAction(Request $request): JsonResponse
         return $this->adminJson(['success' => $success]);
     }
 
-    /**
-     * @Route("/webdav{path}", name="pimcore_admin_webdav", requirements={"path"=".*"})
```

Comment:
What happens if someone used the `name=pimcore_admin_webdav` are you still able to create the route via the name?

Reply:
It is defined in the routing.yaml, it should still work